### DISCUSSION TITLE
Log file for static timing analysis

### DISCUSSION THREAD

**Discussion by DanielDu9453** [bol-edu/HLS-SOC-Discussions#101] [Q&A]
想請問這裡的log檔案是不是就是timing summary輸出出來的文件檔?如果不是，是要如何產生呢?我在synthesis資料夾中似乎沒有看到類似的檔案

**Comment by ZheChen-Bill** [comment]
你會需要在合成前先給予一個clock的constraints，然後synthesis之後再輸出report timing summary

**Reply by DanielDu9453** [reply]
所以log file for static timing analysis就是report timing summary輸出的檔案嗎
